VARIABLES.md

```diff
@@ -2100,6 +2100,662 @@ Default:  "{{control_center_ldap_password}}"
 
 ***
 
+### kafka_connect_replicator_group
+
+Set this variable to customize the Linux Group that the Kafka Connect Replicator Service user belongs to. Default group is confluent.
+
+Default:  "{{kafka_connect_replicator_default_group}}"
+
+***
+
+### kafka_connect_replicator_bootstrap_servers
+
+Variable to define bootstrap servers for Kafka Connect Replicator.  Mandatory for Kafka Connect Replicator setup.
+
+Default:  localhost:9092
+
+***
+
+### kafka_connect_replicator_custom_properties
+
+Use to set custom Kafka Connect Replicator properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+
+Default:  {}
+
+***
+
+### kafka_connect_replicator_cluster_id
+
+Set this variable to customize the Cluster ID used by Kafka Connect Replicator.
+
+Default:  replicator
+
+***
+
+### kafka_connect_replicator_offset_start
+
+Set this variable to customize the offset starting point for Kafka Connect Replicator.
+
+Default:  consumer
+
+***
+
+### kafka_connect_replicator_offsets_topic
+
+Set this variable to customize the topic that Kafka Connect Replicator stores it's offsets in.
+
+Default:  connect-offsets
+
+***
+
+### kafka_connect_replicator_status_topic
+
+Set this variable to customize the topic where Kafka Connect Replicator stores it's status.
+
+Default:  connect-status
+
+***
+
+### kafka_connect_replicator_storage_topic
+
+Set this variable to customize the topic where Kafka Connect Replicator stores it's configuration.
+
+Default:  connect-configs
+
+***
+
+### kafka_connect_replicator_white_list
+
+Set this variable with a comma seperated list of Topics for Kafka Connect Replicator to replicate from.  This is a mandatory variable.
+
+Default:  ""
+
+***
+
+### kafka_connect_replicator_topic_auto_create
+
+Sets if Topics are auto created on the destintation cluster by Kafka Connect Replicator.
+
+Default:  true
+
+***
+
+### kafka_connect_replicator_health_checks_enabled
+
+Boolean to enable health checks on Kafka Connect Replicator.
+
+Default:  true
+
+***
+
+### kafka_connect_replicator_ldap_user
```

Comment:
do these variables need to exist? i think replicator is being deployed without rbac correct?

Reply:
removed in replicator dom branch